lib/avromatic.rb

```diff
@@ -81,13 +82,11 @@ def self.prepare!(skip_clear: false)
   end
 
   def self.eager_load_models=(models)
-    @eager_load_model_names = Array(models).map { |model| model.is_a?(Class) ? model.name : model }
+    @eager_load_model_names = Array.wrap(models).map { |model| model.is_a?(Class) ? model.name : model }
```

Comment:
Was this change necessary? `Array()` and `Array.wrap()` should behave the same here.

Reply:
Not really, it was my OCD kicking in, as I have some fun stories to tell about `Array()` and it calling `#to_ary` first. But I can revert it 😅 .